packages/vite/src/node/optimizer/index.ts

```diff
@@ -33,6 +33,12 @@ export type ExportsData = ReturnType<typeof parse> & {
   hasReExports?: true
 }
 
+export type OptimizedDeps = {
```

Comment:
Any reason why this is a `type` not an `interface`?
There is a typescript-eslint rule for that, should I configure it later?

Reply:
It should be interface, I think we could add the typescript-eslint rule 🙏🏼